test/1_stdlib/Unmanaged.swift

```diff
@@ -54,6 +54,29 @@ UnmanagedTests.test("_withUnsafeGuaranteedRef/return") {
   }
 }
 
+UnmanagedTests.test("toOpaque") {
+  var ref = Foobar()
+  let unknownPtr = unsafeBitCast(
+    Unmanaged.passUnretained(ref).toOpaque(),
+    to: Int.self) 
+  expectNotEqual(0, unknownPtr)
```

Comment:
I'd recommend merging two tests at this point, and calling `fromOpaque(unknownPtr).takeUnretainedValue()` here, and then using `===` to check that the new reference is equal to the old one.